app/components/bd-load-more.js

```diff
@@ -0,0 +1,10 @@
+import Ember from 'ember';
```

Comment:
@arthurra assuming my PR is approved, this is the component that needs some design help. Let me know if you have some time and we can work together or you can knock it out, whichever works for you.